client/components/happychat/style.scss

```diff
@@ -159,6 +159,21 @@
 	animation: happychat-minimize-right .5s 1 forwards;
 }
 
+.layout.has-no-sidebar .happychat .happychat__container.is-open.is-minimizing {
+	animation: happychat-fade-out-right .5s 1 forwards;
+
+	> .happychat__composer,
+	> .happychat__welcome,
+	> .happychat__conversation,
+	> .happychat__title .happychat__active-toolbar * {
+		visibility: visible;
```

Comment:
What do these rules do? I'm a bit concerned with the universal selector.

Reply:
They override the default animation's: https://github.com/Automattic/wp-calypso/blob/2ee42f5b70ace747e7b1ab86da1b9023eb444643/client/components/happychat/style.scss#L143-L154

Reply:
Ah. We def need to ditch the asterisk at some point. It really messes with initial paint time.